Automattic/jetpack
# Boost: Add tracking event when expanding show stopper "learn more" section

## Proposed changes:

* Add tracking event when clicking on the "learn what to do" link to see the steps section. Event is `jetpack_boost_critical_css_learn_more_expanded`. I decided to only track expanding, as we don't need collapse. It's just a similar event and doesn't really help us in any way.

![CleanShot 2024-06-06 at 16 24 11](https://github.com/Automattic/jetpack/assets/11799079/b0b34ad9-8e18-49d8-87d1-eb1f8930bc79)


### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup Boost;
* Add the provided snippet to break all front-end pages;
* Try to regenerate critical/cloud css;
* Click on the "learn what to do" link;
* Check tracks to make sure `jetpack_boost_critical_css_learn_more_expanded` shows up.

Snippet:

```php
<?php

add_action( 'template_redirect', function () {
	exit;
} );
```
